// Copyright 2023 The LUCI Authors. All rights reserved.
// Use of this source code is governed under the Apache License, Version 2.0
// that can be found in the LICENSE file.

syntax = "proto3";

package recipe_modules.recipe_engine.defer.tests;

message CollectInputProps {
  bool exception = 1;
  string step_name = 2;
}

message ContextInputProps {
  bool exception = 1;
  string step_name = 2;
}

message ResultInputProps {
  bool infra_steps = 1;
  bool exception = 2;
  string step_name = 3;
}

message NonDeferredInputProps {
  bool fail = 1;
}

message SuppressedInputProps {
  bool fail = 1;
}
